### DISCUSSION TITLE
Can I setup postfix to use sendgrid smtp?

### DISCUSSION THREAD

**Discussion by whisperdancer** [simple-login/app#1310] [Self-hosting question]
I have a sendgrid account and it's already connected to my domain. All my DNS records for my email are already in place and working. Is it possible to setup postfix to use my sendgrid smtp instead of the way it is described in this Readme.md document? This would solve the ISP port 25 block.

If so, I trust it would require configuration changes to the /etc/postfix/main.cf file to reflect the different configuration. What would the /postfix/main.cf file look like if configured to use sendgrid smtp?

Thank you!

**Comment by nguyenkims** [comment]
@whisperdancer Unfortunately we don't have any experience setting up SL with Sendgrid. I found this page https://docs.sendgrid.com/for-developers/sending-email/postfix, hope it can be helpful.
